spec/unit/form_builder_spec.rb

```diff
@@ -124,6 +124,19 @@ def build_form(options = {}, form_object = Post.new, &block)
     end
   end
 
+  context "file input present" do
+    let :body do
+      build_form do |f|
+        f.input :body, as: :file
+      end
+    end
+    it "adds multipart attribute automatically" do
+      if Rails::VERSION::MAJOR > 3
```

Comment:
The if should wrap the context not only the expectation
And pleas leaf a empty line between the let and it block

BTw: nice to see you back on board!